docs/cmdline-opts/get.md

```diff
@@ -22,7 +22,11 @@ Example:
 When used, this option makes all data specified with --data, --data-binary
 or --data-urlencode to be used in an HTTP GET request instead of the POST
 request that otherwise would be used. The data is appended to the URL
-with a '?' separator.
+with a '?' as a separator. Or with '&' if more data is provided (check
+--data for details).
+
+Prior to 7.86.0 curl could again use '?' as a separator in some
+edge cases even if '?' was already in place.
```

Comment:
This explanation is not helping. What dies it mean to "use it again" as a separator? Separator for what?

If there was a URL parser bug back before 7.86.0, why do we need to explain that to users now?

Reply:
As per **NOK** example from PR description:
```shell
$ curl --silent --get --data key=value 'https://httpbin.org/get?foo/bar=baz' -v
[...]
> GET /get?foo/bar=baz?key=value HTTP/1.1
[...]
```
You can see `?` used twice. So `?` was "used again" by curl, although it was already provided in URL.

Maybe better wording could be:
> Prior to 7.86.0 curl could use '?' as a separator between data fields in some edge cases.

Why to mention it if already fixed? If somebody uses curl prior to 7.86.0 and notices this problem and checks manual, he will switch to newer version instead of creating issue etc.